client/state/action-types.js

```diff
@@ -68,6 +68,8 @@ export const REMOVE_NOTICE = 'REMOVE_NOTICE';
 export const SELECTED_SITE_SET = 'SELECTED_SITE_SET';
 export const SERIALIZE = 'SERIALIZE';
 export const SERVER_DESERIALIZE = 'SERVER_DESERIALIZE';
+export const SET_DOCUMENT_TITLE = 'SET_DOCUMENT_TITLE';
```

Comment:
Acknowledging that inconsistency within this file is a bit misleading, we've started to prefer action types be suffixed with the verb, such that alphabetizing keeps related action types together ([reference](https://github.com/Automattic/wp-calypso/blob/master/docs/our-approach-to-data.md#actions)).